.github/workflows/ci.yml

```
@@ -0,0 +1,26 @@
+name: Continuous Integration
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Run tests
+      run: |
+        npm ci
+        npm run build --if-present
```

```suggestion
        npm run prepare
```

We actually don't have a build script on this repo

---

Dang- good call.